jupyter/datascience/ubi9-python-3.11/Pipfile

```
@@ -7,45 +7,45 @@ name = "pypi"
 
 [packages]
 # Datascience and useful extensions
-boto3 = "~=1.35.14"
-kafka-python-ng = "~=2.2.2"
-kfp = "~=2.9.0"
-matplotlib = "~=3.9.2"
-numpy = "~=2.1.0"
-pandas = "~=2.2.2"
-plotly = "~=5.23.0"
-scikit-learn = "~=1.5.1"
-scipy = "~=1.14.1"
-skl2onnx = "~=1.17.0"
+boto3 = "~=1.37.8"
+kafka-python-ng = "~=2.2.3"
+kfp = "~=2.12.1"
+matplotlib = "~=3.10.1"
+numpy = "~=2.2.3"
+pandas = "~=2.2.3"
+plotly = "~=6.0.0"
+scikit-learn = "~=1.6.1"
+scipy = "~=1.15.2"
+skl2onnx = "~=1.18.0"
 onnxconverter-common = "~=1.13.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
 codeflare-sdk = "~=0.27.0"
 kubeflow-training = "==1.9.0"
 
 # DB connectors
-pymongo = "~=4.8.0"
-psycopg = "~=3.2.1"
-pyodbc = "~=5.1.0"
-mysql-connector-python = "~=9.0.0"
+pymongo = "~=4.11.2"
+psycopg = "~=3.2.5"
+pyodbc = "~=5.2.0"
+mysql-connector-python = "~=9.2.0"
 
 # JupyterLab packages
 
 odh-elyra = "==4.2.0"
 
 jupyterlab = "==4.2.7"
 jupyter-bokeh = "~=4.0.5"
-jupyter-server = "~=2.14.2"
-jupyter-server-proxy = "~=4.3.0"
+jupyter-server = "~=2.15.0"
+jupyter-server-proxy = "~=4.4.0"
 jupyter-server-terminals = "~=0.5.3"
 jupyterlab-git = "~=0.50.1"
```

```suggestion
jupyterlab-git = "~=0.51.0"
```
It seems that there is a new version for this package, can be updated? or we are should stay on the 0.50 because of incompatibility with rest?

---

I believe 0.50.1 was a requirements for jupyterlab ~= 4.2.5, but I will test again quickly to ensure this is accurate, just one moment